packages/main/test/specs/DateTimePicker.spec.js

```diff
@@ -102,6 +102,33 @@ describe("DateTimePicker general interaction", () => {
 		assert.strictEqual(newValue.toUpperCase(), "14/04/2020, 01:02:03 AM", "The new date/time is correctly selected.");
 	});
 
+	it("tests selection of new date without changing the time section", () => {
```

Comment:
This exact test doesn't cover the functionality that is being fixed in this PR. Please create a new test(or edit this one) in order to cover it.

Reply:
This test is testing the behavior of selecting only date, without selecting any time from the TimeSelection part. The problem with reproducing the logged behavior, is that, when initing without correct value, we init with the current time, and this current time will always be different, so the test will either be unstable or more complex. But inspecting the behavior this test should cover the problem.